examples/apps/windows/otAdapter.h

```diff
@@ -344,9 +344,9 @@ public ref class otAdapter sealed
         void set(String^ value)
         {
             char name[OT_NETWORK_NAME_MAX_SIZE + 1];
-            auto err = WideCharToMultiByte(CP_UTF8, WC_NO_BEST_FIT_CHARS, value->Data(), -1, name, ARRAYSIZE(name), nullptr, nullptr);
-            if (err == 0) ThrowOnFailure(otThreadSetNetworkName(DeviceInstance, name));
-            else throw Exception::CreateException(E_INVALIDARG);
+            auto len = WideCharToMultiByte(CP_UTF8, 0, value->Data(), -1, name, ARRAYSIZE(name), nullptr, nullptr);
+            if (len <= 0) throw Exception::CreateException(E_INVALIDARG);
```

Comment:
What is the type of `len`? Is it a signed or unsigned variable? The original code only did a ``err == 0`` here.